nemo_text_processing/inverse_text_normalization/de/taggers/cardinal.py

```diff
@@ -57,9 +57,9 @@ def __init__(self, tn_cardinal_tagger: GraphFst, deterministic: bool = True):
 
         self.graph_ties = (tn_cardinal_tagger.two_digit_non_zero @ optional_delete_space).invert().optimize()
         # this is to make sure if there is an ambiguity with decimal, decimal is chosen, e.g. 1000000 vs. 1 million
-        graph = pynutil.add_weight(graph, weight=0.0001)
+        graph = pynutil.add_weight(graph, weight=0.001)
         self.graph_no_exception = graph
-        self.digit = tn_cardinal_tagger.digit.invert().optimize()
+        self.digit = pynini.arcmap(tn_cardinal_tagger.digit, map_type="rmweight").invert().optimize()
```

Comment:
why do you need "rmweight" here? how do you define the default ITN option if all non-Zero weights with 1?

Reply:
if i don't remove,i cannot do arcsort in the next line.  there is no one to many mapping for ITN, so it will not be a problem here.